arangod/RocksDBEngine/RocksDBVPackIndex.cpp

```diff
@@ -158,11 +158,13 @@ RocksDBVPackIndexIterator::RocksDBVPackIndexIterator(
 
   RocksDBMethods* mthds = RocksDBTransactionState::toMethods(trx);
   rocksdb::ReadOptions options = mthds->iteratorReadOptions();
-  if (!reverse) {
-    // we need to have a pointer to a slice for the upper bound
-    // so we need to assign the slice to an instance variable here
-    _upperBound = _bounds.end();
-    options.iterate_upper_bound = &_upperBound;
+  // we need to have a pointer to a slice for the upper bound
+  // so we need to assign the slice to an instance variable here
+  if (reverse) {
+    _rangeBound = _bounds.start();
```

Comment:
we are only setting `_rangeBound` here, but not using it for anything.
Is that intentional?